### PR TITLE
Release v1.5.3

### PR DESCRIPTION
**Pull request by matthew-white**
This PR prepares the release of v1.5.3. We may release with just the commit from #585, but I thought I'd create a separate branch in case we also want to ship #593 or anything else. This PR only contains commits from other PRs that have already been approved and merged.

### CODE REVIEW

**Comment by matthew-white** [comment]
We're planning to ship a patched version of Slonik that will help with #482 and #485. We also discussed shipping the related #537. (And if we're shipping #537, we should also include #562, because otherwise there'll be a conflict with the `master` branch.)

**Comment by matthew-white** [comment]
> We also discussed shipping the related #537.

@alxndrsn, did I understand that correctly? I'm realizing now that you might have been talking about fixing gajus/slonik#378, not #479.

File: docs/api.md [hunk 1]
```diff
@@ -26,12 +26,18 @@ Forms and their submissions are also accessible through two **open standards spe
...
 * The [OData](http://odata.org/) standard allows data to be shared between platforms for analysis and reporting. Tools like [Microsoft Power BI](https://powerbi.microsoft.com/en-us/) and [Tableau](https://public.tableau.com/en-us/s/) are examples of clients that consume the standard OData format and provide advanced features beyond what we offer. If you are looking for a straightforward JSON output of your data, or you are considering building a visualization or reporting tool, this is your best option.
 
-Finally, **system information and configuration** is available via a set of specialized resources. Currently, you may set the Backups configuration, set the Usage Reporting configuration, and retrieve Server Audit Logs.
+Finally, **system information and configuration** is available via a set of specialized resources. Currently, you may set the set the Usage Reporting configuration and retrieve Server Audit Logs.
```
Comment (yanokwa) [inline docs/api.md:29]: "you may set the set the" should be "you may set the"

Comment (matthew-white) [reply docs/api.md]: Good catch! Should be fixed now!

**Review by lognaturel — APPROVED** [review]

**Comment by matthew-white** [comment]
> This PR only contains commits from other PRs

With the exception of badb3912fdf4d5dca29bd4cd520b9d3b4788db6e. 😅

**Comment by matthew-white** [comment]
v1.5.3 was released off of badb3912fdf4d5dca29bd4cd520b9d3b4788db6e. I resolved conflicts with the `master` branch by opting for the package.json and package-lock.json from the `master` branch, which includes the upgrade to Node 16 from #591. CircleCI is failing because those files don't include `@sinonjs/fake-timers`. @alxndrsn, once I merge this PR, would you mind following up with another PR that reinstalls `@sinonjs/fake-timers` and installs a patched version of Slonik? I imagine you'll need to pull in some of the changes from #599 as well.

**Pull request merged**